llvm/lib/CodeGen/SelectionDAG/DAGCombiner.cpp

```diff
@@ -2005,6 +2006,9 @@ SDValue DAGCombiner::visit(SDNode *N) {
   case ISD::UINT_TO_FP:         return visitUINT_TO_FP(N);
   case ISD::FP_TO_SINT:         return visitFP_TO_SINT(N);
   case ISD::FP_TO_UINT:         return visitFP_TO_UINT(N);
+  case ISD::LRINT:
+  case ISD::LLRINT:
+    return visitXRINT(N);
```

Comment:
Move the `return visitXRINT(N);` to the same column as the other opcodes (i.e.ignore clang-format)

Reply:
Won't the clang-format test fail in the CI, failing the PR?